polaris-service/src/main/java/org/apache/polaris/service/config/PolarisApplicationConfig.java

```diff
@@ -146,6 +149,23 @@ public void setFeatureConfiguration(Map<String, Object> featureConfiguration) {
     this.configurationStore = new DefaultConfigurationStore(featureConfiguration);
   }
 
+  @JsonProperty("maxRequestBodyBytes")
+  public void setMaxRequestBodyBytes(long maxRequestBodyBytes) {
+    if (maxRequestBodyBytes == 0) {
+      // The underlying library that we use to implement the limit treats -1 and 0 as the same, so
+      // we block 0 to prevent ambiguity.
+      throw new IllegalArgumentException(
+          String.format(
+              "maxRequestBodyBytes may not be 0. Use %d to specify no limit.",
```

Comment:
nit: "maxRequestBodyBytes must be a positive integer or `%d` to specify no limit."

Reply:
Updated